server/athenian/api/align/goals/dbaccess.py

```diff
@@ -54,6 +64,49 @@ async def delete_goal(account_id: int, goal_id: int, sdb_conn: Connection) -> No
     await sdb_conn.execute(sa.delete(Goal).where(where_clause))
 
 
+async def delete_team_goals(
+    account_id: int, goal_id: int, team_ids: Sequence[int], sdb_conn: Connection,
+) -> None:
+    """Delete a set of TeamGoal-s from DB."""
+    assert team_ids
+    assert conn_in_transaction(sdb_conn)
+    await _validate_team_goal_deletions(account_id, goal_id, team_ids, sdb_conn)
+
+    where_clause = sa.and_(TeamGoal.goal_id == goal_id, TeamGoal.team_id.in_(team_ids))
+    delete_stmt = sa.delete(TeamGoal).where(where_clause)
+    await sdb_conn.execute(delete_stmt)
+
+
+async def assign_team_goals(
+    account_id: int,
+    goal_id: int,
+    assignments: Sequence[TeamGoalTargetAssignment],
+    sdb_conn: Connection,
+) -> None:
+    """Assign new TeamGoal-s targets for an existing Goal."""
+    await _validate_team_goal_assignments(account_id, goal_id, assignments, sdb_conn)
+
+    now = datetime.now(timezone.utc)
+    values = [
+        {
+            "goal_id": goal_id,
+            "team_id": assign.team_id,
+            "target": assign.target,
+            "created_at": now,
+            "updated_at": now,
```

Comment:
```suggestion
            TeamGoal.goal_id.name: goal_id,
            TeamGoal.team_id.name: assign.team_id,
            TeamGoal.target.name: assign.target,
            TeamGoal.created_at.name: now,
            TeamGoal.updated_at.name: now,
```
We never hardcode column names because it is a nightmare to refactor them otherwise. You can grep the whole codebase for, say, `PullRequest.repository_full_name.name` and you'll get precisely all the usages.